src/library_egl.js

```diff
@@ -110,7 +186,7 @@ var LibraryEGL = {
       EGL.setErrorCode(0x3008 /* EGL_BAD_DISPLAY */);
       return 0;
     }
-    if (config != 62002 /* Magic ID for the only EGLConfig supported by Emscripten */) {
+    if ((config <= 0) || (config > EGL.configAttribs.length)) { /* Check EGLConfig ID is within acceptable range */
```

Comment:
It would be good to do `var config_index = config - 1;` early on here so that the ID to index conversion arithmetic is done exactly once up front, and all other code locations can then index directly and not have to repeat the subtract by one. `config_index < 0 || config_index >= EGL.configAttribs.length` would then look a bit more idiomatic than fusing the subtract one arithmetic into the comparison operators here. Also could store `EGL.configAttribs[config_index]` to a variable to avoid repeating that below.